### DISCUSSION TITLE
[@typescript/KTX2Loader] Import WebGPU in KTX2Loader.d.ts

### DISCUSSION THREAD

**Discussion by riahimedyassin** [DefinitelyTyped/DefinitelyTyped#70307] [Issues with a @types package] [answered]
# Issue 
[@typescript/KTX2Loader]
The import statement of Renderer and WebGPU is not recognized by the package.
Needed to  change it as following in order for the package to work : 
```ts
// Current Version 
import { Renderer, WebGPURenderer } from "three/webgpu";
// Correct Import 
import { Renderer, WebGPURenderer } from "three/build/three.webgpu.js";
```

[a comment by an automated account is not shown]

**Reply by riahimedyassin** [reply]
[@typescript/KTX2Loader]

**Comment by drenaud-solicis** [comment, marked as answer]
@riahimedyassin Hi ! Can you edit the title of this issue please ? :) Bot wasn't detect your tag.

"[three] Import WebGPU in KTX2Loader.d.ts"

**Reply by riahimedyassin** [reply]
I corrected it, I hope it is right now
